genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/v4/AgentConfigRequest.java

```diff
@@ -40,6 +40,7 @@
 @SuppressWarnings("checkstyle:finalclass")
 public class AgentConfigRequest {
     private final boolean archivingDisabled;
+    private final String requestedArchiveLocationPrefix;
```

Comment:
I'm not sure that this is the right class for this data. I understand that the `archivingDisabled` flag is in here and it is distinctly possible that that is in the wrong place as well. It was dumped somewhere temporarily for v3 compatibility.

This is metadata provided by the user which impacts the job and behavior of the job within the overall ecosystem. I might feel that `JobMetadata` is a better place or even a new class entirely like `JobArchiveMetadata` or something. @mprimi thoughts?

Reply:
@mprimi Adding `JobArchiveMetaData` makes sense?

Reply:
No strong preference at this point. Whatever the two of you like is ok with me.